### PR TITLE
Upd templates for Russian and Ukrainian filters

### PR DESCRIPTION
**Pull request by Alex-302**
https://github.com/AdguardTeam/FiltersRegistry/issues/735
https://github.com/AdguardTeam/AdguardFilters/pull/144318

~URLs with `CyrillicFilters` are invalid until https://github.com/AdguardTeam/AdguardFilters/pull/144318 is not merged.~

~TODO: add includes for Belarusian, Bulgarian and Kazakh langs.~

After merge, do not forget about PR in build config in BB.

### CODE REVIEW

File: filters/filter_1_Russian/template.txt [hunk 1]
```diff
@@ -5,32 +5,37 @@
...
+!-------------------------------------------------------------------------------!
+!------------------ Common general hiding rules---------------------------------!
+!-------------------------------------------------------------------------------!
+@include "https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/CyrillicFilters/common-sections/general_elemhide.txt"
+!
 !-------------------------------------------------------------------------------!
 !------------------ Advertising networks ---------------------------------------!
 !-------------------------------------------------------------------------------!
-@include "https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/RussianFilter/sections/adservers.txt"
 @include "https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/RussianFilter/sections/adservers_firstparty.txt"
```
Comment (zloyden) [inline filters/filter_1_Russian/template.txt:25]: Shouldn't there be new includes like this?
```
@include "https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/CyrillicFilters/RussianFilter/sections/adservers_firstparty.txt"
```

**Review by aalpatkina — APPROVED** [review]

**Review by aalpatkina — APPROVED** [review]

File: filters/filter_1_Russian/template.txt (hunk 1 above)
Comment (Alex-302) [reply filters/filter_1_Russian/template.txt]: done

**Review by zloyden — APPROVED** [review]

**Review by slavaleleka — CHANGES_REQUESTED** [review]
update compiler with improved filters_i18n schema (I'll do it)

**Review by aalpatkina — APPROVED** [review]

**Review by aalpatkina — APPROVED** [review]

File: filters/filter_1_Russian/filter.txt [hunk 2]
```diff
@@ -68,6 +68,7 @@
 #%#var AG_setConstant=function(e,a){if("undefined"===a)a=void 0;else if("false"===a)a=!1;else if("true"===a)a=!0;else if("noopFunc"===a)a=function(){};else if("trueFunc"===a)a=function(){return!0};else if("falseFunc"===a)a=function(){return!1};else if(/^\d+$/.test(a)){if(a=parseFloat(a),isNaN(a)||32767<Math.abs(a))return}else return;var b=!1;AG_defineProperty(e,{get:function(){return a},set:function(c){if(b)var d=!0;else void 0!==c&&void 0!==a&&typeof c!==typeof a&&(b=!0),d=b;d&&(a=c)}})};
 ! JS API FINISH
 
+!
```
Comment (zloyden) [inline filters/filter_1_Russian/filter.txt:71]: Is it needed?

**Review by slavaleleka — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Review by AdamWr — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Review by piquark6046 — APPROVED** [review]

**Review by jellizaveta — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**